pkg/volume/volume_types.go

```diff
@@ -108,12 +110,12 @@ func (v *KubernetesVolume) GetVolume(name string) (corev1.Volume, error) {
 	return volume, nil
 }
 
-func (v *KubernetesVolume) ApplyPVCForStatefulSet(containerName string, path string, spec *v1.StatefulSetSpec, meta func(name string) metav1.ObjectMeta) error {
+func (v *KubernetesVolume) ApplyPVCForStatefulSet(containerName string, path string, spec *v1.StatefulSetSpec, meta func(name string, labels, annotations map[string]string) metav1.ObjectMeta) error {
 	if v.PersistentVolumeClaim == nil {
 		return errors.New("PVC definition is missing, unable to apply on statefulset")
 	}
 	pvc := corev1.PersistentVolumeClaim{
```

Comment:
How about something like this to avoid the need to change the signature and force the client to handle this?
```suggestion
	m := meta(v.PersistentVolumeClaim.PersistentVolumeSource.ClaimName)
	labels.Merge(m.Labels, v.PersistentVolumeClaim.Labels)
	labels.Merge(m.Labels, v.PersistentVolumeClaim.Annotations)

	
	pvc := corev1.PersistentVolumeClaim{
	    ObjectMeta: m,
```